Build/Tactical/TeamTurns.cc

```diff
@@ -146,6 +148,9 @@ void StartPlayerTeamTurn( BOOLEAN fDoBattleSnd, BOOLEAN fEnteringCombatMode )
   /// that a baddie can die and still maintain it's attacker ID
   gTacticalStatus.fKilledEnemyOnAttack = FALSE;
 
+  // Save if we are in Dead is Dead mode
+  if (gGameOptions.ubGameSaveMode == DIF_DEAD_IS_DEAD)   DoDeadIsDeadSave();
```

Comment:
Used the wrapper function in GameSettings.h. It didn't change too much, but some headers could be removed. Also fits there as CanGameBeSaved() has it's home there as well.

Also reduces the risk of an accidental DiD Save without polling the mode before (which I found in one instance).